TSPL.docc/LanguageGuide/Protocols.md

```diff
@@ -1368,6 +1421,92 @@ for level in levels.sorted() {
   ```
 -->
 
+## Implicit Conformance to a Protocol
+
+Some protocols are so common that you would write them on almost every type.
+For the following protocols,
+Swift automatically infers the conformance
+when you define a type that implements the protocol's requirements:
+
+- `Codable`
```

Comment:
I was a little surprised to see this here. After checking Swift Evolution I think this implicit conformance only happens for distributed actor, not other struct/class/etc.? If that's the case, some clarification might be beneficial